.github/workflows/link-check.yml

```diff
@@ -1,12 +1,14 @@
-name: Validate link references in built HTML
+name: Check links and link references
 
 on:
   push:
     branches:
       - main
+      - flutter-forward-23
```

Comment:
This can probably go away as I expect this branch name won't be reused

Reply:
Nice catch. Removed :)